gui/dig.lua

```diff
@@ -247,8 +247,13 @@ function GenericOptionsPanel:init()
             end,
             show_tooltip = true,
             on_activate = function()
-                self.dig_panel.placing_extra.active = true
-                self.dig_panel.placing_extra.index = #self.dig_panel.extra_points + 1
+                if self.dig_panel.mark1 and self.dig_panel.mark2 then
+                    self.dig_panel.placing_extra.active = true
+                    self.dig_panel.placing_extra.index = #self.dig_panel.extra_points + 1
+                elseif self.dig_panel.mark1 and not self.dig_panel.mark2 then
+                    local mouse_pos = dfhack.gui.getMousePos()
```

Comment:
be sure to check mouse_pos for nil if the player hits `v` while the mouse is out of bounds